src/dstack/_internal/utils/ssh.py

```diff
@@ -56,8 +56,15 @@ def include_ssh_config(path: PathLike, ssh_config_path: PathLike = default_ssh_c
             with open(ssh_config_path, "r") as f:
                 content = f.read()
         if include not in content:
-            with open(ssh_config_path, "w") as f:
-                f.write(include + content)
+            try:
+                with open(ssh_config_path, "w") as f:
+                    f.write(include + content)
+            except PermissionError:
+                print(f"Couldn't write to {ssh_config_path} due to a permissions problem")
```

Comment:
@peterschmidt85 
We've discussed that a logger is a better option there. But the default logging level is CRITICAL, so we need to change it in `src/dstack/_internal/cli/utils/common.py : configure_logging`